frontend/src/__tests__/cypress/cypress/pages/projects.ts

```diff
@@ -15,6 +17,18 @@ class NotebookRow extends TableRow {
   findOutdatedElyraInfo() {
     return cy.findByTestId('outdated-elyra-info');
   }
+
+  findEnableSwitch() {
+    return this.find().pfSwitch('notebook-status-switch');
+  }
+
+  findNotebookRouteLink() {
+    return this.find().findByTestId('notebook-route-link');
+  }
+
+  findHaveNotebookStatusText() {
```

Comment:
May be we can move `findEnableSwitch` , `findNotebookRouteLink`, `findHaveNotebookStatusText` under `ProjectRow` . Also change this:
```suggestion
  findNotebookStatusText() {
```